src/applications/enrollment-verification/components/EnrollmentVerificationBreadcrumbs.jsx

```diff
@@ -6,7 +6,7 @@ export default function EnrollmentVerificationBreadcrumbs() {
     <a href="/" key="home">
       Home
     </a>,
-    <a href="#" key="education-and-training">
+    <a href="/" key="education-and-training">
```

Comment:
Modified due to eslint error preventing checking.

The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

Reply:
Understood.